easybuild/easyconfigs/p/protobuf-python/protobuf-python-3.10.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'protobuf-python'
+version = '3.10.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/google/protobuf/'
+description = """Python Protocol Buffers runtime library."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com/google/protobuf/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['83a3a0b23ed2b08041dbb0af92d023e5325a4324c2bc8acb55962d349710e52f']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('protobuf', version)
+]
+
+start_dir = 'python'
```

Comment:
@Darkless012 Please also add:

```python
download_dep_fail = True
use_pip = True
```

Reply:
There are some issue with installing it via pip. Need to look closer.

Reply:
installation fails with use_pip = True
the fosscuda 2018b version has it also disabled